pyspecProcScripts/load_data.py

```diff
@@ -385,8 +385,7 @@ def proc_spincore_ODNP_v1(s,fl=None):
     nPoints = s.get_prop('acq_params')['nPoints']
     SW_kHz = s.get_prop('acq_params')['SW_kHz']
     nScans = s.get_prop('acq_params')['nScans']
-    nPhaseSteps = s.get_prop('acq_params')['nPhaseSteps']
-    s.chunk('t',['ph1','t2'],[4,-1])
+    #s.chunk('t',['ph1','t2'],[4,-1])
     s.set_units('t2','s')
```

Comment:
you can't make this change -- it will affect how other files are loaded.

Possibly, you need to use a different version to load your files?  Ask @guinneak198

Reply:
I can't load my files with that line, so yes I would need a different version.